src/main/java/dev/meyi/bn/handlers/ChestTickHandler.java

```diff
@@ -222,4 +223,8 @@ private void orderConfirmation(IInventory chest) {
       }
     }
   }
+  @SubscribeEvent
+  public void renderInChest(GuiScreenEvent.BackgroundDrawnEvent e){
+    BazaarNotifier.modules.drawAll();
+  }
```

Comment:
Do you know if there are any performance drawbacks to the way you've written this now? I don't know if it's any different, but it might be worth pulling out a little bit into this method to speed things up

Like the ones that aren't module-specific shouldn't be in a module-specific `shouldShow` method. Perhaps the module list could have one that's for those global conditions?

What do you think?